### PR TITLE
finished form modal

### PR DESCRIPTION
**Pull request by hennersz**

### CODE REVIEW

File: frontend/src/sass/includes/helpers.scss [hunk 1]
```diff
@@ -66,3 +66,56 @@
...
   }  
 }
+
+.Modal {
+  position: absolute;
+  top: 70px;
+  left: 5vw;
+  right: 5vw;
+  height: 200px;
+	background-color: #fff;
```
Comment (HugoDF) [inline frontend/src/sass/includes/helpers.scss]: Proper indentation mate 😛

File: frontend/src/sass/includes/helpers.scss [hunk 2]
```diff
@@ -66,3 +66,56 @@
...
+
+  @media screen and (min-width: 65em){
+    left: 35vw;
+    right: 35vw;
+  }
+
+}
+
+.Overlay {
+	position: fixed;
```
Comment (HugoDF) [inline frontend/src/sass/includes/helpers.scss]: 4 or 2 space?

**Comment by xoneco** [comment]
I can't get this to work, but I'm most definitely doing something wrong.

**Comment by ChristopherHammond13** [comment]
Done a rebase so this should sort things out

**Pull request merged**